FirebasePerformance/Tests/Unit/Instruments/FPRNSURLSessionInstrumentTest.m

```diff
@@ -381,8 +386,9 @@ - (void)testDelegateURLSessionTaskDidSendBodyDataTotalBytesSentTotalBytesExpecte
   NSURL *URL = [self.testServer.serverURL URLByAppendingPathComponent:@"testUpload"];
   NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:URL];
   request.HTTPMethod = @"POST";
-  NSURL *fileURL = [[NSBundle bundleForClass:[self class]] URLForResource:@"smallDownloadFile"
-                                                            withExtension:@""];
+
+  NSBundle *bundle = [FPRTestUtils getBundle];
```

Comment:
Good catch! This change is not needed because the tests will not run. For consistency, I would prefer to call `getBundle`. Let me know if you have any concerns.